.github/workflows/ci.yml

```diff
@@ -203,3 +203,34 @@ jobs:
         with:
           name: nwaku-master-logs
           path: packages/tests/log/
+
+  maybe-release:
+    name: release
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+    needs: [check, proto, browser, node]
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          monorepo-tags: true
+
+      - uses: actions/checkout@v3
+        if: ${{ steps.release.outputs.release_created }}
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.NODE_JS }}
+          registry-url: "https://registry.npmjs.org"
+        if: ${{ steps.release.outputs.release_created }}
+
+      - run: npm install
+        if: ${{ steps.release.outputs.release_created }}
+
+      - run: npm run build
+        if: ${{ steps.release.outputs.release_created }}
+
+      - run: npm publish --workspaces --next --access public
```

Comment:
I think we can drop approach when we were adding hash to a version considering it will be automatically incremented.
@fryorcraken , what do you think?

Reply:
Can you clarify what strategy is used for incrementing? I think for now we can keep patch increment until we feel confident we reached a good stage on the API, for which we can release `0.1.0`.

From then it'll be more complicated as sometimes we'll patch release and sometimes we'll minor release

Reply:
@fryorcraken this is the algorithm 
```
The most important prefixes you should have in mind are:

fix: which represents bug fixes, and correlates to a [SemVer](https://semver.org/) patch.
feat: which represents a new feature, and correlates to a SemVer minor.
feat!:, or fix!:, refactor!:, etc., which represent a breaking change (indicated by the !) and will result in a SemVer major.
```

I can change the behaviour for version `<1.0.0` so that only `patch` and `minor` will be incremented before we are confident enough. Later we can change this as well.

Reply:
Ok great thanks @weboko.

Looks good. Can we maybe do only `patch` increment for now.
Once https://github.com/waku-org/js-waku/issues/1010 is done we can do `patch` and `minor` increment with `minor` done for breaking changes (`!`.

What does `hidden` mean? does it mean no release?

Reply:
Which `hidden` do you mean?

Reply:
In the default mapping https://github.com/conventional-changelog/conventional-changelog/blob/8076d4666c2a3ea728b95bf1e4e78d4c7189b1dc/packages/conventional-changelog-conventionalcommits/writer-opts.js#L171

```
  config.types = config.types || [
    { type: 'feat', section: 'Features' },
    { type: 'feature', section: 'Features' },
    { type: 'fix', section: 'Bug Fixes' },
    { type: 'perf', section: 'Performance Improvements' },
    { type: 'revert', section: 'Reverts' },
    { type: 'docs', section: 'Documentation', hidden: true },
    { type: 'style', section: 'Styles', hidden: true },
    { type: 'chore', section: 'Miscellaneous Chores', hidden: true },
    { type: 'refactor', section: 'Code Refactoring', hidden: true },
    { type: 'test', section: 'Tests', hidden: true },
    { type: 'build', section: 'Build System', hidden: true },
    { type: 'ci', section: 'Continuous Integration', hidden: true }
  ]
```

Reply:
I guess it hides from the changelog?

Reply:
@fryorcraken 
One of the things I want to make clear as well is that inter dependencies in other packages will be incremented automatically. Before we were using `*` version. Here is an example of the new way. 

![image](https://user-images.githubusercontent.com/118575614/217948214-97c9e198-eefa-49e6-a5ad-06051dc35634.png)

Reply:
Sounds like a safer way to do things